scripts/create_pool/utils/index.ts

```diff
@@ -45,36 +96,39 @@ export const getNativeMintInfo = async function () {
     decimals: 9,
     isInitialized: true,
     freezeAuthority: null,
-    mint: NATIVE_MINT
-  }
-  return nativeMint
-}
+    mint: NATIVE_MINT,
+  };
+  return nativeMint;
+};
 
-export const getTokenMintInfo = async function (ctx: WhirlpoolContext, address: PublicKey) {
+export const getTokenMintInfo = async function (
+  ctx: WhirlpoolContext,
+  address: PublicKey
+) {
   if (address.equals(NATIVE_MINT)) {
-    const nativeMint = await getNativeMintInfo()
-    return nativeMint
+    const nativeMint = await getNativeMintInfo();
+    return nativeMint;
   }
-  const mint = await ctx.fetcher.getMintInfo(address)
-  return mint
-}
+  const mint = await ctx.fetcher.getMintInfo(address);
+  return mint;
+};
 
-export const delay = async function (milliseconds : number) {
-  return new Promise(resolve => setTimeout( resolve, milliseconds))
-}
+export const delay = async function (milliseconds: number) {
+  return new Promise((resolve) => setTimeout(resolve, milliseconds));
+};
 
 export const mapTickSpacing = {
-  1: 'One',
-  8: 'Stable',
-  32: 'ThirtyTwo',
-  64: 'SixtyFour',
-  128: 'Standard'
-}
+  1: "One",
+  8: "Stable",
+  32: "ThirtyTwo",
+  64: "SixtyFour",
+  128: "Standard",
+};
```

Comment:
những cái formatting vầy, nếu được thì mình nên để nó trong PR khác chỉ toàn là formatting thôi (không liên quan đến logic) thì sẽ tốt hơn nhen em ơi